src/ddmd/codegenASTFamily.d

```diff
@@ -0,0 +1,56 @@
+module ddmd.codegenASTFamily;
+
+import ddmd.arraytypes;
+import ddmd.dclass;
+import ddmd.denum;
+import ddmd.dimport;
+import ddmd.dstruct;
+import ddmd.expression;
+import ddmd.globals;
+import ddmd.identifier;
+import ddmd.mtype;
+
+struct CodegenASTFamily
+{
+    alias EnumDeclaration = .EnumDeclaration;
```

Comment:
I see you updated this as I was reviewing. Taking this one step further: move all imports inside this struct.